packages/block-library/src/navigation/edit/edit-menu-toolbar-button.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BlockControls,
+} from '@wordpress/block-editor';
+import { useSelect, useDispatch } from '@wordpress/data';
+import {
+	ToolbarButton,
+	ToolbarGroup,
+} from '@wordpress/components';
+import { store as interfaceStore } from '@wordpress/interface';
+import { __ } from '@wordpress/i18n';
+
+const EditMenuToolbarButton = () => {
+ 	// Blocks can be loaded into both post and site editors.
+	// We use this to determine which editor we are in so that
+	// we can determine which inspector controls to open.
+	const {
+		isPostEditor,
+	} = useSelect(
+		( select ) => {
+			// eslint-disable-next-line @wordpress/data-no-store-string-literals
+			const editorSelectors = select( 'core/editor' );
+			return {
+				isPostEditor: !! editorSelectors.getEditedPostAttribute( 'type' )
+			};
+		}
+	);
+	const { enableComplementaryArea } = useDispatch( interfaceStore );
```

Comment:
I'd probably avoid importing the store here as well as this will bundle the interface package into block-library and that package is only meant to be used in top level packages (edit-post, edit-site, edit-widgets...)